tsconfig.json

```diff
@@ -1,13 +1,12 @@
 {
   "compilerOptions": {
+    "resolveJsonModule": true,
+    "moduleResolution": "node",
     "outDir": "./dist/",
-    "module": "es6",
-    "target": "es6",
+    "module": "commonjs",
+    "target": "es5",
```

Comment:
I had to change theses following this docs: https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-9.html#new---resolvejsonmodule.
I checked the output. I didn't find anything different in the final output. It's just moving the work of the rollup plugin bubble to typescript

Reply:
Are you sure about the change in `module` and `target`?

(did not find anything in the doc and it's working on my machine while keeping `es6`)

Reply:
yes you're right. I simply followed their doc but that part doesn't seem necessary